enclave_manager/setup.py

```diff
@@ -56,8 +59,10 @@
     enclave_bridge_wrapper_path,
     enclave_bridge_path,
     os.path.join(tcf_root_dir, 'common/cpp'),
+    os.path.join(tcf_root_dir, 'tc/sgx/trusted_worker_manager/common'),
```

Comment:
Why is there a dependency on `tc/sgx/trusted_worker_manager/common`

Reply:
Because work_order_wrap.cpp depends on work_order.h (enclave bridge file) which uses header files in tc/sgx/trusted_worker_manager/common.